tests/functional/healthchecks/test/checkRoutes.js

```diff
@@ -0,0 +1,54 @@
+const assert = require('assert');
+const http = require('http');
+const https = require('https');
+const conf = require('../../config.json');
+
+const transportStr = conf.transport;
+const ipAddress = conf.ipAddress;
+
+const transport = transportStr === 'http' ? http : https;
+
+function checkResult(expectedStatus, res) {
+    const actualStatus = res.statusCode;
+    assert.strictEqual(actualStatus, expectedStatus);
+}
+
+function makeChecker(expectedStatus, done) {
+    return res => {
+        checkResult(expectedStatus, res);
+        done();
+    };
+}
+
+describe('Healthcheck routes', () => {
+    it('should return 200 OK on GET request', done => {
```

Comment:
Just for clarification, do you mean the it statements themselves or the code in them?